modules/ROOT/pages/fcct-config.adoc

```diff
@@ -1,53 +0,0 @@
-= Provisioning and Configuration
```

Comment:
Maybe we should leave a stub page or redirect so we don't break existing outside links?

Reply:
Stub added with some explanatory text.